parse.py

```diff
@@ -35,30 +44,30 @@
     find_latest_update,
     get_item_modifiers,
     get_item_modifiers_by_text,
+    get_item_modifiers_by_id,
     get_leagues,
     get_ninja_bases,
     query_item,
 )
 from utils.web import open_trade_site, wiki_lookup
 
-DEBUG = False
-
-
 def parse_item_info(text: str) -> Item:
     """
     Parse item info (from clipboard, as obtained by pressing Ctrl+C hovering an item in-game).
     """
-    # TODO: test if poe item
+    # TODO: test if poe item (??)
```

Comment:
Parse_item_info receives all text captured by the clipboard not just poe items. An idea is to move this parse function into the Item class as a class function and do the poe item check in the clipboard function.

Reply:
check len(item_list) > 2 and throw exception, catch in calling methods